### ISSUE TITLE
Exporting Image Stuck on Loading 

### ISSUE THREAD

**Issue by whoisemiliano** [chocoford/ExcalidrawZ#5]
Steps to Reproduce : 

1. Download the latest version 
2. Create a new document
3. Add some items to it
4. Try to export the image

It gets stuck on a loading screen 

<img width="422" alt="image" src="https://github.com/chocoford/ExcalidrawZ/assets/26259799/37b6787d-d103-41ee-976e-1840ff6b3592">

**Comment by chocoford** [comment]
Hello @whoisemiliano. Thank you for your feedback. We will carefully review the functionality and fix the issue as soon as possible!

**Comment by whoisemiliano** [comment]
Thanks @chocoford 🚀 and thanks for building a cool mac app for this, this is my new go-to diagram app for everything :D

**Comment by chocoford** [comment]
After investigation, we have identified the issue: **attempting to export an image from an empty document causes the function to fail in subsequent operations**. We have fixed this bug, but due to changes in [Firebase's policy](https://firebase.google.com/support/faq#hosting-exe-restrictions), we must find a new hosting service provider to support the software update function. This may take a few days. For now, simply restarting the software will restore the image export function to normal, and *NO NOT* exporting images from an empty document.
